### PR TITLE
For connoisseur only: add a simple "ephyviewer" backend plot_traces().

### PR DESCRIPTION
**Pull request by samuelgarcia**
For connoisseur only: add a simple "ephyviewer" backend plot_traces().

1. no add in pyproject.toml because this imply pyside or pyqt hich is enormous.
2. no CI testing because Qt

I test it locally and seams to work.

@grahamfindlay I promise I will have a look to this very soon so we will also have the mode='map': 
https://github.com/NeuralEnsemble/ephyviewer/pull/177

```python
rec, sorting = si.generate_ground_truth_recording(num_channels=32, durations=[3600.,], seed=225)
si.plot_traces(rec, segment_index=0, backend="ephyviewer", mode="line", show_channel_ids=True)
```

Screenshot_20230919_174450

### CODE REVIEW

**Comment by alejoe91** [comment]
Cool! Can we add this to the docs too? I think it's very useful!!

Here: https://spikeinterface.readthedocs.io/en/latest/modules/widgets.html#usage

Also, pre-commit pls ;)

**Comment by samuelgarcia** [comment]
> Cool! Can we add this to the docs too? I think it's very useful!!

Good idea.
Done

File: doc/modules/widgets.rst [hunk 1]
```diff
@@ -85,6 +88,28 @@ Finally, if you wish to set up another cloud provider, follow the instruction fr
...
+
+
+For pip based install, run:
+
+.. code-block:: bash
+
+    pip install PySide6 ephyviewer
+
+
+Anaconda user will have a better experience with this:
```
Comment (alejoe91) [inline doc/modules/widgets.rst:104]:
```suggestion
Anaconda users will have a better experience with this:
```

File: doc/modules/widgets.rst [hunk 2]
```diff
@@ -215,6 +240,21 @@ For example, here is how to combine the timeseries and sorting summary generated
     print(url)
 
 
+ephyviewer
+^^^^^^^^^^
+
+
+The :code:`ephyviewer` backend is only available for :py:func:`~spikeinterface.widgets.plot_traces()` functions.
```
Comment (alejoe91) [inline doc/modules/widgets.rst:247]:
```suggestion
The :code:`ephyviewer` backend is currently only available for the :py:func:`~spikeinterface.widgets.plot_traces()` function.
```

**Review by alejoe91 — APPROVED** [review]

**Review by alejoe91 — APPROVED** [review]

**Comment by alejoe91** [comment]
Nice! Merging!

**Pull request merged**